taccsite_cms/static/site_cms/css/src/site.blog.css

```diff
@@ -1,10 +0,0 @@
-/* DO NOT ADD STYLES HERE; ONLY IMPORT OTHER STYLESHEETS */
-
-/* NOTE: This file exists so that the Blog/News styles can be loaded only if feature is enabled */
-
-/* Organize via ITCSS */
-/* SEE: https://confluence.tacc.utexas.edu/x/IAA9Cw */
-
-/* COMPONENTS */
-@import url("_imports/components/django.cms.blog.css");
-@import url("_imports/components/django.cms.post.css");
```

Comment:
**Explanation.** This file is replaced by `app.blog.css` (akin to `app.taccsite_system_specs.css`).